src/bsoncxx/array/view.hpp

```diff
@@ -25,6 +25,13 @@
 
 namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
+
+namespace types {
+namespace bson_value {
+class view;
+}  // namespace bson_value
+}  // namespace types
+
```

Comment:
Is this used somewhere?

Reply:
This forward declaration is necessary in order to make the `find_value` declaration below compile.